test/e2e/network_test.go

```diff
@@ -554,12 +577,14 @@ func (tc *testContext) waitUntilJobSucceeds(name string) error {
 		}
 		if job.Status.Failed > 0 {
 			events, _ := tc.getPodEvents(name)
-			return errors.Errorf("job %v failed: %v", job, events)
+			logs, _ := tc.getLogs(job)
```

Comment:
shouldnt we capture the error we get here?

Reply:
Good idea. I'll add that in